src/Aspire.Hosting.Testing/DistributedApplicationTestingBuilder.cs

```diff
@@ -107,6 +109,32 @@ public static async Task<IDistributedApplicationTestingBuilder> CreateAsync(Type
         return await factory.CreateBuilderAsync(cancellationToken).ConfigureAwait(false);
     }
 
+    /// <summary>
+    /// Creates a new instance of <see cref="IDistributedApplicationTestingBuilder"/>.
+    /// </summary>
+    /// <param name="args">The command line arguments to pass to the entry point.</param>
+    /// <returns>
+    /// A new instance of <see cref="IDistributedApplicationTestingBuilder"/>.
+    /// </returns>
+    public static IDistributedApplicationTestingBuilder Create(params string[] args)
+        => Create(args, (_, __) => { });
+
+    /// <summary>
+    /// Creates a new instance of <see cref="IDistributedApplicationTestingBuilder"/>.
+    /// </summary>
+    /// <param name="args">The command line arguments to pass to the entry point.</param>
+    /// <param name="configureBuilder">The delegate used to configure the builder.</param>
+    /// <returns>
+    /// A new instance of <see cref="IDistributedApplicationTestingBuilder"/>.
+    /// </returns>
+    public static IDistributedApplicationTestingBuilder Create(string[] args, Action<DistributedApplicationOptions, HostApplicationBuilderSettings> configureBuilder)
```

Comment:
YESS!

Reply:
😍😍😍😍  I was just about to write an issue asking for such an API.

THANK YOU!.